.github/workflows/master_build_pr.yml

```diff
@@ -14,126 +14,52 @@ jobs:
     if: ${{ inputs.type == 'ui-build' }}
     runs-on: ui-build
     env:
-      NEXUS_REPO_NPM: tessell-repos-npm-development
-      NEXUS_PULL_REPOS_M2: tessell-m2-development
+      CI-BRANCH: TDEVOPS-1832
```

Comment:
Use `CI_BRANCH` (_ instead of -) following the naming convention of other variables.